ray-curator/ray_curator/tasks/__init__.py

```diff
@@ -0,0 +1,6 @@
+from .document import DocumentBatch
+from .file_group import FileGroupTask
+from .image import ImageBatch, ImageObject
+from .tasks import EmptyTask, Task, _EmptyTask  # TODO: maybe this results in circular imports?
```

Comment:
Has this been checked?

Reply:
Yes, I believe ayush has just copied over from our internal repo, and the example is working there